job-service-db/src/main/resources/v3.3.0/changelog.xml

```diff
@@ -23,10 +23,11 @@
                                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
                                        http://www.liquibase.org/xml/ns/dbchangelog
                                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
-    <changeSet id="add_identity_column" runOnChange="true" author="Gamma Team">
-        <addColumn schemaName="public" tableName="job">
-            <column name="identity" type="SERIAL">
-                <constraints nullable="false" />
+
+    <changeSet id="update_job_task_data_add_suspended" runOnChange="true" author="Gamma Team">
```

Comment:
It worries me that this line is still not identical to the way it is in v3.3.  Hashes are involved here so there needs to be a good reason for it not to be identical.